force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,24 +418,50 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
-        
+        private String name;
+        private Integer age;
         // Constructor to initialize the name and age
-    
+        public Person(String name,Integer year)
+        {
+            //this.name = name;
+            setName(name);
+            //this.age = year;
+            setAge(age);
+        }
         // Method introduceYourself that returns the details of the person        
         // Return a string in the format "Hello, my name is <name> and I am <age> years old."
         // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
-
+        public string introduceYourself()
+        {
+            return 'Hello, my name is ' + this.getName() + ' and I am ' + this.getAge() + ' years old.';
```

Comment:
Since the name and age instance variable already exist, you don't need to call the method inside your code:
```suggestion
            return 'Hello, my name is ' + this.name + ' and I am ' + this.age + ' years old.';
```